src/NzbDrone.Core/ImportLists/MyAnimeList/MalImport.cs

```diff
@@ -0,0 +1,227 @@
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Security.Cryptography;
+using Newtonsoft.Json;
+using NLog;
+using NzbDrone.Common.Http;
+using NzbDrone.Common.Serializer;
+using NzbDrone.Core.Configuration;
+using NzbDrone.Core.Parser;
+using NzbDrone.Core.Parser.Model;
+
+namespace NzbDrone.Core.ImportLists.MyAnimeList
+{
+    public class MalImport : HttpImportListBase<MalListSettings>
+    {
+        public const string OAuthUrl = "https://myanimelist.net/v1/oauth2/authorize";
+        public const string OAuthTokenUrl = "https://myanimelist.net/v1/oauth2/token";
+        public const string RedirectUri = "http://localhost:8989/oauth.html";
```

Comment:
```suggestion
        public const string RedirectUri = "https://auth.servarr.com/v1/mal_sonarr/auth";
        public const string RenewUri = "https://auth.servarr.com/v1/mal_sonarr/renew";
        public const string ClientId = "402de1a90f8eb545f625739fa69ddb98";
```

Reply:
I setup an App for Sonarr on MAL and had it setup on the Servarr auth side of things.